docs/process/Estimation-Process.md

```diff
@@ -27,73 +27,130 @@ below the estimate. These assumptions translate to risks - because if
 
-This approach is clearly a **gross simplification of reality**, and so
-we need to be very careful not to produce garbage estimates.
+Note that it's impossible (in general) to predict all the things that could go
+wrong with a task. This is why estimates are estimates, and not commitments.
+Pilots and co-pilots should feel free to give estimates based on their personal
+experience. They may use their accumulated knowledge of how long similar tasks
+have taken in the past to develop new estimates.
 
-### Points
+The **deviation** between the most-probable estimate and most-pessimistic
+estimate allows the authors of each estimate to express their degree of
+uncertainty in the estimate, and it allows people viewing the estimates to gain
+insight into that uncertainty.
 
-| Points | Size                                            |
-| -----: | ----------------------------------------------- |
-|      1 |                                                 |
-|      2 |                                                 |
-|      3 |                                                 |
-|      5 |                                                 |
-|      8 |                                                 |
-|     13 |                                                 |
-|   > 13 | This ticket needs to be split up further.       |
+A very large deviation between the most-probable and most-pessimistic estimate
+tells us that the author is very uncertain in their estimate. We might use
+this information to take action, for example, by breaking up a task into
+smaller, more well-defined pieces.
 
-### Each _Task_ ticket has an estimate
+It's important to realize that:
 
-We assign an estimation to every single task ticket. If there's no
-estimate, then it can't be added to the sprint.
+- These estimates pertain to the number of days of remaining effort required to
+  perform that task **in isolation**. If two or more tasks are performed in
+  parallel, with context switching, then we need to keep the additional cost of
+  that context switching in mind.
 
-### Ticket must be ready to estimate before estimation
+- These estimates refer to a number of days **remaining**, and as such they
+  should be regularly updated as circumstances change, rather that set in stone
+  and left unchanged.
 
-If there is not enough information about a task ticket, then the
-estimation will be rubbish. So before estimation starts we need:
+## Guidelines for creating estimates
 
-1. The _Task_ ticket is linked to a _User Story_ ticket.
-2. The ticket description contains adequate information about what is
-   expected to be done.
-3. The ticket includes acceptance criteria.
-4. The ticket is sufficiently small to estimate.
-5. Task dependencies are defined with jira issue links.
+- Estimates should only be created and updated by the pilot and co-pilot
+  responsible for a given task.
 
-## Team Estimation
+- Estimates should be created only after a _Task_ ticket has entered the
```

Comment:
I like this a lot. We should only consider adding tasks to a sprint that are in a READY state. But is this process documented anywhere else? If so, could we add a link to it? I haven't heard about the READY state before.

Reply:
Yes, we should add a new page with our definition of ready, as it applies to estimation for task tickets.